ci/rt.sh

```diff
@@ -208,8 +209,8 @@ elif [ $mac = d -o $mac = c ]; then #for WCOSS2
 fi
 
 if [[ "$compiler" == MISSING ]] ; then
-    if [[ "$machine" == URSA ]] ; then
-	usage FATAL ERROR: You must specify the compiler on Ursa: -C 'intel|intelllvm' 1>&2
+    if [[ "$machine" != HERA -a "$machine" != WCOSS2]] ; then
```

Comment:
@gspetro-NOAA This change causes failure on Hera with missing compiler option. 
```suggestion
   if [[ "$machine" == "URSA" || "$machine" == "ORION" || "$machine" == "HERCULES" ]]; then
```